File: user-rest-service-2.0.0-client-php-codegen_develop-2.0/src/main/java/io/swagger/client/model/ShowcaseDatatypeMath.java
package io.swagger.client.model;



import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;



@ApiModel(description = "")
public class ShowcaseDatatypeMath  {
  
  private Double bigDecimal = null;
  private Long bigInteger = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("bigDecimal")
  public Double getBigDecimal() {
    return bigDecimal;
  }
  public void setBigDecimal(Double bigDecimal) {
    this.bigDecimal = bigDecimal;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("bigInteger")
  public Long getBigInteger() {
    return bigInteger;
  }
  public void setBigInteger(Long bigInteger) {
    this.bigInteger = bigInteger;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ShowcaseDatatypeMath {\n");
    
    sb.append("  bigDecimal: ").append(bigDecimal).append("\n");
    sb.append("  bigInteger: ").append(bigInteger).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: user-rest-service-1.0.2/generated-code/csharp/src/main/csharp/io/swagger/Model/ShowcaseDatatypeMath.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;



namespace io.swagger.Model {
  public class ShowcaseDatatypeMath {
    

    
    public long? BigInteger { get; set; }

    

    
    public double? BigDecimal { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ShowcaseDatatypeMath {\n");
      
      sb.Append("  BigInteger: ").Append(BigInteger).Append("\n");
      
      sb.Append("  BigDecimal: ").Append(BigDecimal).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}